utils/Dockerfile.blueprint

```diff
@@ -266,6 +266,7 @@ COPY utils/startup_commands.php /usr/local/bin/startup_commands.php
 COPY utils/enable_apache_mods.php /usr/local/bin/enable_apache_mods.php
 {{end}}
 COPY utils/docker-entrypoint.sh /usr/local/bin/docker-entrypoint.sh
+COPY utils/apache-expose-envvars.sh /usr/local/bin/apache-expose-envvars.sh
```

Comment:
This should be copied only for Apache variants. So you can simply move this line 2 lines up.